docs/user_guide/gui_workflow.rst

```diff
@@ -56,19 +59,21 @@ only after you know the API.
    #. Click once on the qubit.
    #. Click ``Shift+F2``
    #. Double-click on the qubit.
-   #. In the new *Object Properties* window locate a tab *PCell
+   #. In the new *Instance Properties* window locate a tab *PCell
       parameters*.
-   #. Scroll down in the list of parameters to locate *Coupler lengths*.
-   #. Change the value in the text box to ``100,160,0``.
+   #. Scroll down in the list of parameters to locate *Include coupler between qubits*.
+   #. Click on the box.
```

Comment:
Looks like in this part you changed it so that it changes a parameter of the Demo chip (`Include coupler between qubits`) instead of `Coupler lengths` parameter of a single qubit. It shouldn't be possible to change the demo chip parameters at this point if it was turned to static as explained in earlier step of the tutorial. Probably the confusion was because you didn't turn Demo chip to static and because the selection of the qubit does not work easily by double clicking. I suggest changing this part of the tutorial to the following:

```
#. **Disable a coupling port for the qubit.**

   #. From the *Tools* panel select *Select*.
   #. Drag a selection box around the qubit.
   #. Click ``Shift+F2``
   #. Press ``q``.
   #. In the new *Instance Properties* window locate a tab *PCell
      parameters*.
   #. Scroll down in the list of parameters to locate *Coupler lengths*.
   #. Change the value in the text box to ``100,160,0``.
   #. Click ``Ok``.
```

Please check if this makes sense to you and change accordingly.